api/resource/factory_test.go

```diff
@@ -9,10 +9,12 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+
```

Comment:
why a newline here? usually we keep the github imports together with the k8s.io imports

Reply:
Sure. I have a different goimport grouping setup.